libclc/generic/lib/integer/ctz.cl

```diff
@@ -0,0 +1,42 @@
+#include "../clcmacro.h"
```

Comment:
Supplement the vector types supported by the function through the gentype.inc file.